src/studio-home/card-item/index.tsx

```diff
@@ -60,7 +61,11 @@ const CardItem: React.FC<Props> = ({
     courseCreatorStatus,
     rerunCreatorStatus,
   } = useSelector(getStudioHomeData);
-  const destinationUrl: string = path ?? new URL(url, getConfig().STUDIO_BASE_URL).toString();
+  const waffleFlags = useSelector(getWaffleFlags);
+
+  const destinationUrl: string = waffleFlags.useNewCourseOutlinePage
+    ? path ?? url
+    : path ?? new URL(url, getConfig().STUDIO_BASE_URL).toString();
```

Comment:
This is not very readable.  It seems to me that the waffle flag should be checked only if `path` is `null` or `undefined`.  So maybe pivot the outer conditional around that.

Reply:
[inform] Implemented in follow-up commit: [`5f19746` (#1372)](https://github.com/openedx/frontend-app-authoring/pull/1372/commits/5f19746fd6b2a803200abd8abdaac6adb5813fb2#diff-c364658aa941a905003b93c1c4d55585292cb684f305c7c9de16365976615ae5R66)